qoqo_for_braket/qoqo_for_braket/backend.py

```diff
@@ -613,3 +679,51 @@ def run_measurement_queued(self, measurement: Any) -> QueuedProgramRun:
 
             queued_circuits.append(self.run_circuit_queued(run_circuit))
         return QueuedProgramRun(measurement, queued_circuits)
+
+    def run_program_queued(
+        self, program: QuantumProgram, params_values: List[List[float]], hybrid: bool = False
+    ) -> List[Union[QueuedProgramRun, QueuedHybridRun]]:
+        """Run a qoqo quantum program on a AWS backend multiple times return a list of queued Jobs.
+
+        This effectively performs the same operations as `run_program` but returns
+        queued results.
+
+        The hybrid parameter can specify whether to run the program in hybrid mode or not.
+
+        Args:
+            program (QuantumProgram): the qoqo quantum program to run.
+            params_values (List[List[float]]): the parameters values to pass to the quantum
+                program.
+            hybrid (bool): whether to run the program in hybrid mode.
+
+        Raises:
+            ValueError: incorrect length of params_values compared to program's input
+                parameter names.
+
+        Returns:
+            List[Union[QueuedProgramRun, QueuedHybridRun]]
+        """
+        queued_runs: List[Union[QueuedProgramRun, QueuedHybridRun]] = []
+        input_parameter_names = program.input_parameter_names()
```

Comment:
You haven't added this check to the function above, should it be?

Reply:
what check exactly?

Reply:
The check below for the input_parameter_names length

Reply:
Ah ok, I had to replicate what `program.run()` or `program.run_register()` did, so check -> substituting params -> calling `run_measurement(_...)`. In the upper method everything is handled by that call automatically, here I needed to call the queued versions.